src/main/java/net/runelite/rsb/internal/globval/GlobalWidgetInfo.java

```diff
@@ -121,17 +155,17 @@ public enum GlobalWidgetInfo {
      * [EQUIPMENT] Widget Info
      */
     EQUIPMENT_ITEMS_CONTAINER(WidgetIndices.WornEquipmentTab.GROUP_INDEX, WidgetIndices.WornEquipmentTab.PARENT_CONTAINER),
-    EQUIPMENT_HEAD(WidgetIndices.WornEquipmentTab.GROUP_INDEX,  WidgetIndices.WornEquipmentTab.HEAD_DYNAMIC_CONTAINER),
-    EQUIPMENT_BACK(WidgetIndices.WornEquipmentTab.GROUP_INDEX,  WidgetIndices.WornEquipmentTab.BACK_DYNAMIC_CONTAINER),
-    EQUIPMENT_NECK(WidgetIndices.WornEquipmentTab.GROUP_INDEX,  WidgetIndices.WornEquipmentTab.NECK_DYNAMIC_CONTAINER),
+    EQUIPMENT_HELMET(WidgetIndices.WornEquipmentTab.GROUP_INDEX,  WidgetIndices.WornEquipmentTab.HELMET_DYNAMIC_CONTAINER),
```

Comment:
how have you came to conclude that these slot names a proper ones?

I vote against this change as equipment slots shouldn't be named regarding what they use.

Reply:
I don't know what are the proper ones, but I think it is best to follow Runelite's conventions, given we are a Runelite wrapper/bot anyway.
https://static.runelite.net/api/runelite-api/net/runelite/api/EquipmentInventorySlot.html

Reply:
NO! Runelite can serve as great example of how things should never be done.

Many times I had to correct either incorrect naming or entirely incorrect data from Runelite.

Reply:
the names were not uniform across the board.  i took a look at what open-osrs did for these and copied the naming from there.

the bank-worn-equipment widget names were not uniform with the worn-equipment names.
this was just an attempt to get 'em both on the same page.